File: 코딩테스트 입문/Javascript/이어 붙인 수.js
function solution(num_list) {
    let answer = 0;
    let [odd_sum,even_sum] = ["",""];
    
    num_list.forEach((v)=>{
        if(v%2===0){
            even_sum += v.toString();
        }
        else{
            odd_sum += v.toString();
        }
    })
    
    answer = Number(odd_sum) + Number(even_sum);
    
    return answer;
}

File: 코딩테스트 입문/Python/이어 붙인 수.py
def solution(num_list):
    answer = 0
    odd_list = ""
    even_list = ""
    
    for i in num_list:
        if i%2==0:
            even_list += str(i)
        else:
            odd_list += str(i)
    
    answer = int(even_list) + int(odd_list)
    
    return answer